### ISSUE TITLE
"md-alt-text" does not show in Firefox

### ISSUE THREAD

**Issue by eassa** [angular/material#9771]
``` html
<span class="md-alt-text">.... </span>
```

``` css
.md-button span.md-alt-text{
    -webkit-box-flex:0;
    -webkit-flex-grow:0;
    flex-grow:0;
    -webkit-align-self:flex-end;
    align-self:flex-end;
    margin:0 8px
}
```

does not show in the toolbar in edge

**Comment by clshortfuse** [comment]
What you are experiencing is a Firefox bug related to child elements of buttons (md-icon) do not receive mouseover/hover events. 

https://bugzilla.mozilla.org/show_bug.cgi?id=843003

md-tooltip should be a direct child of md-button as shown in the demos: https://material.angularjs.org/latest/demo/tooltip

**Comment by clshortfuse** [comment]
This happens because Firefox does not have support for flex styled buttons:

It appears as though a fix scheduled for m52: 

https://bugzilla.mozilla.org/show_bug.cgi?id=984869

In the meantime, we need to make sure we wrap the spans inside a div set to flex.

**Comment by eassa** [comment]
@clshortfuse  it does not work on angular material design demos page either !! , if you go to "Menu Bar" and click on the ( file , edit ...) menu.  in firefox it wont show you the hotkeys (which is were this class are used and my problem with it ) , while in chrome and edge it shows

**Comment by clshortfuse** [comment]
@eassa Yes, I am aware. If I wasn't clear, when I meant, in the meantime,  I meant to say this needs to be fixed before waiting for Firefox 52.0
